airbyte-cdk/python/airbyte_cdk/sources/streams/core.py

```diff
@@ -73,38 +75,48 @@ def state(self, value: MutableMapping[str, Any]) -> None:
         """State setter, accept state serialized by state getter."""
 
 
-class Stream(ABC):
+class Stream(AbstractStream, ABC):
     """
     Base abstract class for an Airbyte Stream. Makes no assumption of the Stream's underlying transport protocol.
     """
 
+    def read(
```

Comment:
I'm completely open to you responding with "tests will slow us down at this stage of iteration since we're still figuring out what the interfaces should even be" , but just noting that this doesn't have tests and it probably should once we're happy with the structure